src/NServiceBus.Core/Timeout/Hosting/Windows/TimeoutDispatcherProcessor.cs

```diff
@@ -32,7 +32,16 @@ public bool Handle(TransportMessage message)
 
             if (TimeoutsPersister.TryRemove(timeoutId, out timeoutData))
             {
-                MessageSender.Send(timeoutData.ToTransportMessage(), timeoutData.ToSendOptions(Configure.LocalAddress));
+                try
+                {
+                    MessageSender.Send(timeoutData.ToTransportMessage(), timeoutData.ToSendOptions(Configure.LocalAddress));
+                }
+                catch
+                {
+                    timeoutData.Id = null;
+                    timeoutData.Time = DateTime.UtcNow.AddSeconds(5);
```

Comment:
@andreasohlund @SeanFeldman In discussion with @timbussmann this is the line that raised most concerns:
"timeoutData.Time = DateTime.UtcNow.AddSeconds(5)"
Do you think it's ok? I don't like having some hardcoded magic number and resetting the timeout for another future time, but this is the only way we found to make it work.